docs/reference/operators/ebpf.md

```diff
@@ -0,0 +1,34 @@
+---
+title: ebpf
+sidebar_position: 10
+---
+
+The eBPF operator handles layer of type
+`application/vnd.gadget.ebpf.program.v1+binary`, i.e. layers that contain an
+ELF file with the eBPF programs. This operator loads the programs into the
+kernel and attaches them to the different hooks as specified by the gadget
+developer.
```

Comment:
What about formatters: enum, bitfield, kstack?

IMO, adding a TODO is fine for this PR. It's impossible to cover everything in one single PR.

Reply:
Added a "Document operator implementation" TODO in the description as well.